main.py

```diff
@@ -345,12 +347,7 @@ def dummy():
     return "OK"
 
 
-@app.route('/realtime/give-bool', methods=['GET'])
-def give_bool():
-    return str(randint(0, 1))
-
-
-@app.route('/realtime/data', methods=['GET'])
+@app.route('/realtime/data1', methods=['GET'])
```

Comment:
Why is the endpoint data1 now? realtime.js is still requesting `/realtime/data`.